app/src/main/res/values/colors.xml

```diff
@@ -1,6 +1,12 @@
-<?xml version="1.0" encoding="utf-8"?>
+<!--?xml version="1.0" encoding="UTF-8"?-->
```

Comment:
Why was this commented out?  Otherwise LGTM.

Reply:
I think the that line would not affect  so i commented out in case it's needed

Reply:
Isn't it what makes an XML a valid one?

Reply:
i think it's not necessary because other .xml file in values doesn't include that too

Reply:
Wot why comment it then?